source/graphql/traits.d

```diff
@@ -467,3 +457,92 @@ template PossibleTypesImpl(Type, AllTypes...) {
 		}
 	}
 }
+
+// compiler has a hard time inferring safe. So we have to tag it.
+@safe
+void execForAllTypes(T, alias fn, Context...)(auto ref Context context)
+{
+	// establish a seen array to ensure no infinite recursion.
+	execForAllTypesImpl!(T, fn)((bool[void*]).init, context);
+}
+
+@trusted private void *keyFor(TypeInfo ti)
```

Comment:
the * goes on the type not on the function name.

Reply:
OK